nlp/admin/server/src/main/kotlin/fr/vsct/tock/nlp/admin/AdminVerticle.kt

```diff
@@ -358,6 +358,18 @@ open class AdminVerticle : WebVerticle() {
             }
         }
 
+        blockingJsonGet("/logs/intent/stats/:name")
```

Comment:
-> use a POST as /logs/stats does instead of a GET. Here you always use the default locale (!)